dev-client/.prettierrc

```diff
@@ -1,7 +1,25 @@
 {
   "arrowParens": "avoid",
+  "singleQuote": true,
+  "semi": true,
```

Comment:
Questions: This means "anyplace that could have a semicolon should have a semicolon", right?

Reply:
Yes. We're already doing that. I added it to the file so the prettier configuration better matches the web client.